### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by iNoahNothing**
- Remove install scripts
- Update CRDs to getambassador.io/v2
- Retest everything so it still works
- Update configs that rotted

(Should have done this in more than one commit oops)

Signed-off-by: Noah Krause <krausenoah@gmail.com>

### CODE REVIEW

**Comment by iNoahNothing** [comment]
TODO: Update README.md to state renewed purpose

File: api-auth-with-github/README.md [hunk 1]
```diff
@@ -2,21 +2,21 @@
 
 This will enable API authentication using GitHub login credentials. It uses [Keycloak](https://www.keycloak.org/) to act as an OpenID Connect authentication server.
 
-1. Install Ambassador Pro: `make apply-ambassador`.
-2. Create an OAuth application in GitHub.
+1. [Install Ambassador Pro](https://www.getambassador.io/user-guide/install).
```
Comment (kflynn) [inline api-auth-with-github/README.md]: Ambassador Pro?

File: ratelimit/README.md [hunk 2]
```diff
@@ -2,73 +2,50 @@
 
 Ambassador Pro includes a dynamic rate limiting service that can handle a wide variety of rate limiting use cases. 
 
-## Global Rate Limiting
+## Prerequesites
 
-Suppose we want to limit a certain subset of users to only 10 requests per minute for any requests through Ambassador. We can configure a global rate limit that can rate limit based off a header that identifies this subset of users. Users with the header `x-limited-user: true` will be limited to 10 requests per minute. 
+1. [Install Ambassador Edge Stack](https://www.getambassador.io/early-access/user-guide/getting-started/)
```
Comment (kflynn) [inline ratelimit/README.md]: Should this reference the EA URL or the real URL?

**Review by kflynn — CHANGES_REQUESTED** [review]
Two questions, I think it looks OK otherwise? I'm trusting Noah to have actually followed the docs in here. 🙂

File: ratelimit/README.md (hunk 2 above)
Comment (iNoahNothing) [reply ratelimit/README.md]: Probably real. Thanks!

File: api-auth-with-github/README.md (hunk 1 above)
Comment (iNoahNothing) [reply api-auth-with-github/README.md]: Nope! Thanks!

File: env.sh.example [hunk 3]
```diff
@@ -1,24 +1,9 @@
...
-AMBASSADOR_ADMIN_URL="http://ambassador-admin.default.svc.cluster.local:8877"
-# Set this to the address that Ambassador is internally reachable from
-AMBASSADOR_INTERNAL_URL="https://ambassador.default.svc.cluster.local"
+## Example OAuth Authentication with Github settings
+##
+CLIENT_ID=5a3691b3d059dbf98095
+CLIENT_SECRET=49cd212341c82f797502720bf9ec9d0bb11676d6
 
+## Configuration settings for monitoring configuration
+AMBASSADOR_URL=aes.nkrause.k736.net
```
Comment (khussey) [inline env.sh.example]: @nbkrause is it OK that your name is here?

File: grpc-web/ambassador-module.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: getambassador.io/v2
+kind: Module
+metadata:
+  name: ambassador
+spec:
+  config:
+    enable_grpc_web: True
```
Comment (khussey) [inline grpc-web/ambassador-module.yaml]: Is the special character after True here OK?

File: ratelimit/quote-backend-mapping.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  name: quote-backend
+  namespace: ambassador
+spec:
+  prefix: /backend/
+  service: quote
+  labels:
+    ambassador:
+      - remote_address_label_group:
+        - remote_address
```
Comment (khussey) [inline ratelimit/quote-backend-mapping.yaml]: Is the special character at the end here OK?

File: ratelimit/rl-per-user.yaml [hunk 6]
```diff
@@ -5,6 +5,6 @@ metadata:
 spec:
   domain: ambassador
   limits:
-   - pattern: [{x_limited_user: "false"}, {remote_address: "*"}]
+   - pattern: [{remote_address: "*"}]
      rate: 20
      unit: minute
```
Comment (khussey) [inline ratelimit/rl-per-user.yaml]: Another special character here?

**Review by khussey — COMMENTED** [review]
@nbkrause I made some comments, not sure if they are real concerns. Otherwise I am +1.

**Review by kflynn — APPROVED** [review]
Looks OK I think.

**Review by khussey — APPROVED** [review]
+1

**Comment by LukeShu** [comment]
Why the removal of `tracing/`?

**Comment by LukeShu** [comment]
Also: Nice job! This is *waaaay* more thorough  than I would have been.

**Review by LukeShu — APPROVED** [review]

**Pull request merged**